content/en/docs/installation/_index.en.md

```diff
@@ -221,6 +221,32 @@ this windows executable: [fission.exe](https://github.com/fission/fission/releas
 {{< /tab >}}
 {{< /tabs >}}
 
+
+## Verify fission installation
+Once you are done with fission installation, run these commands to make sure fission is installed successfully and all core components of fission are working properly.
+
+To check fission is installed successfully, run this command and verify both client and server version should be same. 
+
+```shell
+fission version
+```
+
+To check fission core components are working properly, run this command.
+
+```shell
+fission check
+```
+
+{{% notice info %}}
+If you have enabled authentication while installing fission, mentioned above commands won't show proper result. You need to generate token to make it work.
+
+See [How to generate auth token](https://fission.io/docs/installation/authentication/#generating-auth-token), if authentication is enabled.
+{{% /notice %}}
+
+{{< notice warning >}}
+If any of these commands failed to run or not showing result properly, please consider re-intalling of fission.
```

Comment:
Let's remove this warning. It doesn't make sense to reinstall Fission if its a Fission issue.

Reply:
ok.
But we should add something in place of this warning. Most of the time i have seen user reaching out to us saying that fission is not working, and when we ask the output of `fission-version` or `fission-check` command then it doesn't work for them, and then user reinstall fission to make it work.